src/Service/ReceiptHelper.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\CaseDocumentRelation;
+use App\Entity\CaseEntity;
+use App\Entity\DigitalPost;
+use App\Entity\HearingPost;
+use Doctrine\Bundle\DoctrineBundle\EventSubscriber\EventSubscriberInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Events;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ReceiptHelper implements EventSubscriberInterface
+{
+    public function __construct(private MailTemplateHelper $mailTemplateHelper, private DocumentUploader $documentUploader, private EntityManagerInterface $entityManager, private DigitalPostHelper $digitalPostHelper, private TranslatorInterface $translator)
+    {
+    }
+
+    public function getSubscribedEvents()
+    {
+        return [
+            Events::postPersist,
+        ];
+    }
+
+    public function postPersist(LifecycleEventArgs $args): void
+    {
+        $entity = $args->getEntity();
+
+        $case = null;
+        $template = null;
+        if ($entity instanceof CaseEntity) {
+            $case = $entity;
+            $digitalPostRecipients[] = (new DigitalPost\Recipient())
+                ->setName($case->getBringer())
+                ->setIdentifierType($case->getBringerIdentification()->getType())
+                ->setIdentifier($case->getBringerIdentification()->getIdentifier())
+                ->setAddress($case->getBringerAddress())
+            ;
+            $template = $case->getBoard()?->getReceiptCase();
+            $documentTitle = $this->translator->trans('Case receipt', [], 'case');
+            // @fixme What should this be? Can we get it from somewhere?
+            $documentType = 'Case receipt';
```

Comment:
So far we have just set these to fitting values (See [AgendaInspectionController](https://github.com/itk-dev/naevnssekretariatet/blob/8169341f4576c18e400728b001b58b682cfcba62/src/Controller/AgendaInspectionController.php#L75) and [AgendaBroadcastController](https://github.com/itk-dev/naevnssekretariatet/blob/8169341f4576c18e400728b001b58b682cfcba62/src/Controller/AgendaBroadcastController.php#L98)) and then used the translation trick in [templates/mail_template.html.twig](https://github.com/itk-dev/naevnssekretariatet/blob/develop/templates/translations/mail_template.html.twig) to ensure translation.